### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by KurochkaR**
upd syncronizer catch wrong phone

### CODE REVIEW

**Review by OlegBondarUA — APPROVED** [review]

File: scripts/new_uklon_daily.py [hunk 1]
```diff
@@ -1,4 +1,5 @@
-from auto.tasks import get_driver_reshuffles
+from auto.tasks import get_driver_reshuffles, check_available_fleets
+
 
 def run(*args):
-    get_driver_reshuffles.delay()
\ No newline at end of file
+    print(check_available_fleets(4))
```
Comment (SpaceIgor) [inline scripts/new_uklon_daily.py:5]: rm

File: auto_bot/handlers/order/handlers.py [hunk 2]
```diff
@@ -485,9 +487,11 @@ def client_reject_order(update, context):
             else:
                 duty.duty += cancel - total_sum
                 report_for_client = have_duty
+                
         duty.save()
         text_to_client(order=order,
                        text=report_for_client)
+                       '''
```
Comment (SpaceIgor) [inline auto_bot/handlers/order/handlers.py:494]: why, its work

File: scripts/new_uklon_daily.py (hunk 1 above)
Comment (KurochkaR) [reply scripts/new_uklon_daily.py:5]: don't check useless scripts

File: auto_bot/handlers/order/handlers.py (hunk 2 above)
Comment (KurochkaR) [reply auto_bot/handlers/order/handlers.py:494]: coment for testing after that i got error in portmone after checking i'll uncomment

**Pull request merged**